htdocs/societe/card.php

```diff
@@ -63,6 +65,12 @@
 	require_once DOL_DOCUMENT_ROOT.'/eventorganization/class/conferenceorboothattendee.class.php';
 }
 
+if ($mysoc->country_code == 'GR') {
+	$u = $conf->global->AADE_WEBSERVICE_USER;
```

Comment:
`$conf->global->xxx `
is now deprecated. Use instead
`getDolGlobalString('xxx');`